diff --git a/node_modules/isomorphic-git/index.cjs b/node_modules/isomorphic-git/index.cjs
index 9f54c74..61f5140 100644
--- a/node_modules/isomorphic-git/index.cjs
+++ b/node_modules/isomorphic-git/index.cjs
@@ -16742,8 +16742,10 @@ var index = {
   writeTag,
   writeTree,
   stash,
+  GitTree
 };
 
+exports.GitTree = GitTree;
 exports.Errors = Errors;
 exports.STAGE = STAGE;
 exports.TREE = TREE;
diff --git a/node_modules/isomorphic-git/index.js b/node_modules/isomorphic-git/index.js
index eaf7498..7e15091 100644
--- a/node_modules/isomorphic-git/index.js
+++ b/node_modules/isomorphic-git/index.js
@@ -16736,7 +16736,8 @@ var index = {
   writeTag,
   writeTree,
   stash,
+  GitTree
 };
 
 export default index;
-export { Errors, STAGE, TREE, WORKDIR, abortMerge, add, addNote, addRemote, annotatedTag, branch, checkout, clone, commit, currentBranch, deleteBranch, deleteRef, deleteRemote, deleteTag, expandOid, expandRef, fastForward, fetch, findMergeBase, findRoot, getConfig, getConfigAll, getRemoteInfo, getRemoteInfo2, hashBlob, indexPack, init, isDescendent, isIgnored, listBranches, listFiles, listNotes, listRefs, listRemotes, listServerRefs, listTags, log, merge, packObjects, pull, push, readBlob, readCommit, readNote, readObject, readTag, readTree, remove, removeNote, renameBranch, resetIndex, resolveRef, setConfig, stash, status, statusMatrix, tag, updateIndex$1 as updateIndex, version, walk, writeBlob, writeCommit, writeObject, writeRef, writeTag, writeTree };
+export { GitTree, Errors, STAGE, TREE, WORKDIR, abortMerge, add, addNote, addRemote, annotatedTag, branch, checkout, clone, commit, currentBranch, deleteBranch, deleteRef, deleteRemote, deleteTag, expandOid, expandRef, fastForward, fetch, findMergeBase, findRoot, getConfig, getConfigAll, getRemoteInfo, getRemoteInfo2, hashBlob, indexPack, init, isDescendent, isIgnored, listBranches, listFiles, listNotes, listRefs, listRemotes, listServerRefs, listTags, log, merge, packObjects, pull, push, readBlob, readCommit, readNote, readObject, readTag, readTree, remove, removeNote, renameBranch, resetIndex, resolveRef, setConfig, stash, status, statusMatrix, tag, updateIndex$1 as updateIndex, version, walk, writeBlob, writeCommit, writeObject, writeRef, writeTag, writeTree };
